### PR TITLE
FIX: added script in package.json of docs and added instructions to run nextra-DocsGPT locally

### PR DESCRIPTION
**Pull request by shelar1423**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
fix and docs update
- **Why was this change needed?** (You can also link to an open issue here)
To setup  Nextra-DocsGPT docs  locally 

- **Other information**:
addd missing script in package.json:
image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# nextra-docsgpt
\ No newline at end of file
+# nextra-docsgpt
+
+## Setting Up Docs Folder of DocsGPT Locally
+
+### 1. Clone the DocsGPT repository:
+
+commandline
```
Comment (dartpain) [inline docs/README.md:7]: Good PR just please remove commandline word
Instead do 
```bash
git clone https://github.com/arc53/DocsGPT.git
```

**Review by dartpain — CHANGES_REQUESTED** [review]
Good PR just please remove commandline word
Instead do
```bash
git clone https://github.com/arc53/DocsGPT.git

**Comment by shelar1423** [comment]
@dartpain i have already added ``` so i guess no need for bash , what say ?

**Comment by shelar1423** [comment]
done @dartpain

**Review by dartpain — APPROVED** [review]

**Pull request merged**